charts/common/tests/deployment_test.yaml

```diff
@@ -1,87 +1,47 @@
-probes: &probes
-      probes:
-        liveness:
-          path: "/actuator/health/liveness"
-        readiness:
-          path: "/actuator/health/readiness"
-        startup:
-          failureThreshold: 300
-          periodSeconds: 1
-
-values: &values
-  env: dev
-  app: testsuite
-  shortname: tstsut
-  team: common
-  ingress:
-    host: test.dev.entur.io
-    trafficType: public
-  labels:
-    custom: label
-  deployment:
-    labels:
-      version: 1
-  containers:
-    - name: test
-      image: img
-      env:
-        - FOO: bar
-      <<: *probes
-
-
 suite: test deployment
+values:
+  - ./values/common-test-values.yaml
 templates:
   - deployment.yaml
 tests:
   - it: must have labels
```

Comment:
To make the asserting clear I think it's better to define the values you are asserting in the set stanza:
´´´
    set:
      env: dev
      labels:
        custom: label
´´´

Reply:
The labels are tested with nearly every template test. So makes sense to put them in the common values file.

Reply:
Why not both? This specific test case is asserting that these values are present. There's great value in having these values visibly set in the testcase. For all other tests these values need to be present but they're not being tested.